docs/internal/release-procedure.md

```diff
@@ -0,0 +1,99 @@
+# Release Procedure
+
+## 🗺️ Merge localization sync PR
+
+A pull request titled "**Update translation files**" is submitted periodically by the "**github-actions**" bot to pull in the localization data from [**Transifex**](https://www.transifex.com/arduino-1/ide2/dashboard/).
+
+If there is an open PR, this must be merged before making the release.
+
+It will be shown in these search results:
+
+https://github.com/arduino/arduino-ide/pulls/app%2Fgithub-actions
+
+## ⚙ Create the release on GitHub
+
+First of all, you need to **set the new version in all the `package.json` files** across the app (`./package.json`, `./arduino-ide-extension/package.json`, `./browser-app/package.json`, `./electron-app/package.json`), create a PR, and merge it on the `main` branch.
+
+To do so, you can make use of the `update:version` script.
+
+For example, if you want to release the version `<YOUR_VERSION>`, you should run the following commands:
+
+```text
+git checkout main
+git pull
+git checkout -b version-<YOUR_VERSION>
+yarn update:version <YOUR_VERSION>
+git commit -am <YOUR_VERSION>
+git push origin version-<YOUR_VERSION>
+```
+
+replacing `<YOUR_VERSION>` with the version you want to release. Then create a PR and merge it.
+
+Then, you need to **create and push the new tag** and wait for the release to appear on [the "**Releases**" page](https://github.com/arduino/arduino-ide/releases).
+
+⚠ Doing this will create a new release and users who already have the IDE installed will be notified from the automatic updater that a new version is available. Do not push the tag if you don't want that.
+
+```text
+git checkout main
+git pull
+git tag -a <YOUR_VERSION> -m "Version <YOUR_VERSION>"
```

Comment:
`git tag -a <YOUR_VERSION> -m "<YOUR_VERSION>"` as the example?

All previous tags have the same message as the version, but I am also OK with this.

Reply:
I see it has been done every which way (`git tag <YOUR_VERSION>`, `git tag -a <YOUR_VERSION> -m "<YOUR_VERSION>"`, `git tag -a <YOUR_VERSION> -m "Version <YOUR_VERSION>"`):
```text
$ git tag --list -n 2.0.0-*                                               
2.0.0-beta.1    2.0.0-beta.1                                              
2.0.0-beta.10                                                             
2.0.0-beta.11                                                             
2.0.0-beta.12                                                             
2.0.0-beta.2    2.0.0-beta.2                                              
2.0.0-beta.3    2.0.0-beta.3                                              
2.0.0-beta.4    2.0.0-beta.4                                              
2.0.0-beta.5    2.0.0-beta.5                                              
2.0.0-beta.6                                                              
2.0.0-beta.7                                                              
2.0.0-beta.8                                                              
2.0.0-beta.9                                                              
2.0.0-rc1       Update version to 2.0.0-rc1                               
2.0.0-rc2       Version 2.0.0-rc2                                         
2.0.0-rc3                                                                 
2.0.0-rc4       Version 2.0.0-rc4                                         
2.0.0-rc5       Version 2.0.0-rc5                                         
2.0.0-rc6       ignore workspace root check in changelog workflow (#960)  
2.0.0-rc7       Version 2.0.0-rc7                                         
2.0.0-rc8       2.0.0-rc8                                                 
2.0.0-rc9       Version 2.0.0-rc9                                         
2.0.0-rc9.1     Version 2.0.0-rc9.1                                       
2.0.0-rc9.2     Version 2.0.0-rc9.2                                       
2.0.0-rc9.3     2.0.0-rc9.3                                               
```

I prefer `git tag -a <YOUR_VERSION> -m "<YOUR_VERSION>"` because this is the way it is done in the other Tooling Team projects (e.g., Arduino CLI, Arduino Create Agent, Arduino Firmware Uploader, Arduino Lint), but it isn't a strong preference.

I just copied this verbatim from the internal document.

@AlbyIanna what do you think?

Reply:
sounds good to me 👍

Reply:
Fixed by https://github.com/arduino/arduino-ide/pull/1414/commits/5ed7596e5686133561d9b5ebd5bb882284ed1865